### PR TITLE
Xfail normalization tests

### LINKED ISSUE #4404: CI failing test_auto_normalize_collection
The CI on master is currently failing. The bad test is `distributed/tests/test_client.py::test_auto_normalize_collection`.

The problem appears to be in `dask/dask`. Running a `git bisect` shows that this has been failing since dask/dask#7016.

With dask and distributed `2020.12.0` the test passes. But with `dask@master` all distributed versions that I've tested fail.

### PR DESCRIPTION
**Pull request by jacobtomlinson**
Closes #4404

Alternative to dask/dask#7035

### CODE REVIEW

**Comment by jakirkham** [comment]
cc @mrocklin

File: distributed/tests/test_client.py [hunk 1]
```diff
@@ -4355,6 +4355,7 @@ def test_normalize_collection_with_released_futures(c):
 
 
 @gen_cluster(client=True)
+@pytest.mark.xfail(reason="https://github.com/dask/distributed/issues/4404")
```
Comment (jrbourbeau) [inline distributed/tests/test_client.py:4358]: It looks like we need to switch the order of `@pytest.mark.xfail` and `@gen_cluster` in order for the `xfail` to take affect

Comment (jakirkham) [inline distributed/tests/test_client.py:4358]:
```suggestion
@pytest.mark.xfail(reason="https://github.com/dask/distributed/issues/4404")
@gen_cluster(client=True)
```

Comment (jakirkham) [reply distributed/tests/test_client.py:4357]: Thanks James! Fixed below 😄

**Review by jrbourbeau — APPROVED** [review]
Thanks @jacobtomlinson!

**Comment by jacobtomlinson** [comment]
Thanks both for nudging this!

**Pull request merged**